src/styles/stars.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  display: flex;
+  height: 16px;
+  justify-content: space-between;
+  margin-bottom: 24px;
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+
+    font-size: 10px;
+    font-weight: 400;
+    color: #060b35;
+    line-height: 14px;
+  }
+
+  &__star {
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+    width: 16px;
+    margin-right: 4px;
+  }
+
+  &__4 {
+    display: flex;
+  }
+}
+
+.stars__4 .stars__star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
Your stars should have functionality from `layout-stars` task, if we want to set 3 instead of 4, it should show 3 active stars, now it will show 0